pkg/windows/multi-minion.ps1

```diff
@@ -0,0 +1,363 @@
+<#
+.SYNOPSIS
+Script for setting up an additional salt-minion on a machine with Salt installed
+
+.DESCRIPTION
+This script will configure an additional minion on a machine that already has a
+Salt installation using one of the Salt packages. It will set up the directory
+structure required by Salt. It will also lay down a minion config to be used
+by the Salt minion. Additionaly, this script can start the new minion in a
+hidden window.
```

Comment:
**Suggestion (spelling, blocking):** "Additionally."

**Suggestion (phrasing, style, non-blocking):** If you want to follow the Google style preference to keep this in the present tense, I recommend this rewrite:

```
This script configures an additional minion on a machine that already has a
Salt installation using one of the Salt packages. It sets up the directory
structure required by Salt. It also lays down a minion config to be used
by the Salt minion. Additionally, this script can start the new minion in a
hidden window.
```